tools/mtmd/mtmd.cpp

```diff
@@ -1024,9 +1024,6 @@ const char * mtmd_image_tokens_get_id(const mtmd_image_tokens * image_tokens) {
 }
 
 llama_pos mtmd_image_tokens_get_n_pos(const mtmd_image_tokens * image_tokens) {
-    if (image_tokens->use_mrope_pos) {
-        return 1; // for M-RoPE, the whole image is 1 in temporal dimension
-    }
```

Comment:
So based on what discussed so far, here we should return `max(h, w)`

However, I don't know how it will impact the rest of the code. Make sure to test it.